src/platform/forms/save-in-progress/SaveStatus.jsx

```diff
@@ -38,10 +47,13 @@ function SaveStatus({
   return (
     <div>
       {autoSavedStatus === SAVE_STATUSES.success && (
-        <div className="panel saved-success-container">
-          <i className="fa fa-check-circle saved-success-icon" />
-          {appSavedSuccessfullyMessage}
-          {savedAtMessage}
+        <div className="panel saved-success-container vads-u-display--flex">
+          <i className="fa fa-check-circle saved-success-icon vads-u-margin-top--0p5" />
+          <div>
```

Comment:
Is this extra `<div>` here for a styling reason?

Reply:
yes, previously only the `<i>` was indented, so the second line would wrap and content would appear under the icon. I added `vads-u-display--flex` to the wrapping `<div>` so the `<i>` is fixed in a left aligned column and the `<div>` you see here contains all the text in a right-aligned column.

Reply:
This is what it would look like without the flex and the `<div>` wrapping the text.

![Screen Shot 2020-11-18 at 4 18 50 PM](https://user-images.githubusercontent.com/136959/99595153-ddb13980-29b9-11eb-951c-95dcc6e872ae.png)

Reply:
@bkjohnson Any chance you have time to finish this review? Thanks!

Reply:
I don't have any objections, but it isn't clear to me if this is an approved design or not, since it would affect all SiP forms applications. I'll follow up again.

Reply:
Hey @bkjohnson - I am the designer who created this design that @Mottie has implemented. I am on a VFS team, so not platform, but @peggygannon has taken a look and so we should be all set. Would you like me to tag Crystabel or Emily in here to get some sign off? This is a rather small change only affecting SiP status so we didn't feel it demanded an entire process. But let me know how you'd like to proceed - thanks!

Reply:
Hey @kevinstachura! Because this will affect all forms in production which use save in progress, we wanted to make sure it got the VSP design and content sign off.

I agree that it's small enough to warrant a quick turnaround. If you have suggestions for what that process might look like, please reach out in slack on #vsp-design-system! We're still figuring out how best to get these kinds of contributions into the system, balancing development speed and platform stability.